me.go

```diff
@@ -1,27 +1,40 @@
 package control
 
+// The Me struct represents data accociated with your Ably account.
 type Me struct {
+	// The access token used to authenticate.
 	Token   Token   `json:"token"`
 	User    User    `json:"user"`
 	Account Account `json:"account"`
 }
 
+// An access token used to authenticate with the Control API.
 type Token struct {
-	ID           string   `json:"id"`
-	Name         string   `json:"name"`
+	// The ID of the token.
+	ID string `json:"id"`
+	// The name of the token.
+	Name string `json:"name"`
+	// The capabilities of the token.
 	Capabilities []string `json:"capabilities"`
 }
 
+// The user component of an Ably account.
 type User struct {
-	ID    int    `json:"id"`
+	// The ID of the user.
+	ID int `json:"id"`
+	// The user's email address.
 	Email string `json:"email"`
 }
 
+// The account detials of an Ably account.
 type Account struct {
-	ID   string `json:"id"`
+	// The ID of the account.
+	ID string `json:"id"`
+	// The name of the account.
 	Name string `json:"name"`
 }
 
+// Me fetches Ably account info.
```

Comment:
"fetches information about the [token](https://ably.com/documentation/control-api#authentication) the current user authenticates with.", not "data associated with account"